docs/openapi/schemas.yaml

```diff
@@ -1475,3 +1475,48 @@ BrokenBacklink:
     url_suggested:
       description: The URL suggested for redirecting from the broken backlink.
       $ref: '#/URL'
+ImportAssistantRequest:
+  type: object
+  required:
+    - url
+    - command
+  properties:
+    url:
+      description: The URL to analyze
+      $ref: '#/URL'
+    command:
+      description: Assistant command to execute.
+      type: string
+    prompt:
+      description: Optional prompt to provide context for the analysis.
+      type: string
+    options:
+      description: Optional configuration options required by the command.
+      type: object
+  example:
+    - url: 'https://example.com/en/page1'
+      command: 'start'
+    - url: 'https://example.com/en/page2'
+      command: 'cleanup'
+      prompt: 'breadcrumbs'
+ImportAssistantResponse:
+  type: object
+  properties:
+    choices:
+      description: The choices available from the assistant command.
+      type: array
+      items:
+        type: object
+        properties:
+          finish_reason:
```

Comment:
The API convention looks to be camelCase for properties, is there a reason to go with snake_case instead?

Reply:
I would be in favour of massaging it to meet how we want the API to look. After all, we may sub out the underlying model if another option comes along.